libamiibo/Data/AmiiboTag.cs

```diff
@@ -20,25 +20,29 @@
  * THE SOFTWARE.
  */
 
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
 using LibAmiibo.Data.Figurine;
 using LibAmiibo.Data.Settings;
 using LibAmiibo.Data.Settings.AppData;
+using LibAmiibo.Data.Settings.AppData.Games;
 using LibAmiibo.Helper;
 using Org.BouncyCastle.Asn1;
 using Org.BouncyCastle.Asn1.Sec;
 using Org.BouncyCastle.Crypto.Parameters;
 using Org.BouncyCastle.Math;
 using Org.BouncyCastle.Security;
-using LibAmiibo.Data.Settings.AppData.Games;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace LibAmiibo.Data
 {
     public class AmiiboTag
     {
+        private static readonly byte[] tagHeader = new byte[] { 0x04, 0x01, 0x02, 0x8F, 0x03, 0x04, 0x05, 0x06, 0x04, 0x48, 0x0f, 0xe0, 0xf1, 0x10, 0xff, 0xee, 0xa5, 0x00, 0x00, 0x00 };
```

Comment:
Can you split this up with comments for better overview?

Header:
9 bytes are NTAG serial, one byte internal, 2 bytes LockBytes, 4 bytes CC, 1 byte magic, 2 bytes write count, 1 byte figure version

Footer:
4 bytes dynamic lock bytes, 4 bytes CFG0, 4 bytes CFG1

Reply:
Also please don't mix uppercase and lowercase hex notation. The code base uses uppercase hex.